docker-for-ibm-cloud/load-balancer.md

```diff
@@ -62,7 +62,17 @@ Use the load balancer to access [DTR](/datacenter/dtr/2.4/guides/).
    **Tip**: Your user name is `admin` or the user name that your admin created for you. You got the password when you [created the cluster](administering-swarms.md#create-swarms) or when your admin created your credentials.
 
 ## Service load balancer
-When you create a service, any ports that are opened with `-p` are automatically exposed through the load balancer. For example:
+
+When you create a service, any ports that are opened with `-p` are automatically exposed through the load balancer.
```

Comment:
or `--publish`

Reply:
Keep in mind that `EXPOSE` is a docs-only thing. You are actually publishing ports, not just exposing them. This may impact the next section too.